pyproject.toml

```
@@ -51,7 +51,7 @@ fsspec = { version = ">= 2023.4.0", optional = true}
 gcsfs = { version = ">= 2023.4.0", optional = true }
 s3fs = { version = ">= 2023.4.0", optional = true }
 adlfs = { version = ">= 2023.4.0", optional = true }
-kfp = { version = ">= 1.8.19", optional = true }
+kfp = { version = ">= 1.8.19, <= 1.8.22", optional = true }
```

```suggestion
kfp = { version = ">= 1.8.19, < 2", optional = true }
```

Can we put the upper-bound at the major version instead?